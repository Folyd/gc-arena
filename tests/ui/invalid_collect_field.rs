use gc_arena::Collect;

#[derive(Collect)]
#[collect(no_drop)]
struct MyStruct {
    #[collect(invalid_arg)] field: u8
}

fn main() {}
