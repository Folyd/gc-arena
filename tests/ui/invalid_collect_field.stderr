error: Only `#[collect(require_static)]` is supported on a field
 --> $DIR/invalid_collect_field.rs:6:15
  |
6 |     #[collect(invalid_arg)] field: u8
  |               ^^^^^^^^^^^
